src/olympia/abuse/tests/test_models.py

```diff
@@ -1252,6 +1252,35 @@ def test_process_decision_escalate_addon_action(self):
         assert new_job.target_addon == addon
         assert report.reload().cinder_job == new_job
 
+    def test_process_decision_for_legal_reviewed_job(self):
+        """An add-on forwarded to the legal queue for review will be a job that may not
+        contain any attached abuse reports (i.e. if an add-on was forwared without a
+        job)."""
+        target = addon_factory()
+        cinder_job = CinderJob.objects.create(job_id='1234', target_addon=target)
```

Comment:
Is it possible to explicitly set "None" to the values that we are testing are in fact None.

That's probably the default but it helps clarify the intentions of the test and protects against any weird scenario where those values are defined internally.

Reply:
the fields aren't on `CinderJob` - they're on `AbuseReport` and `ContentDecision` - it's that there aren't any `AbuseReport`s or `ContentDecision`s (both have a foreign key to the `CinderJob`)

Reply:
It's impossible for that assertion ever to fail, but as requested https://github.com/mozilla/addons-server/pull/23024/commits/c88399c50fb1b9dcdd9291a8b38d5f25c21f77f6